src/commands/up.rs

```diff
@@ -90,6 +90,10 @@ pub struct UpCommand {
     /// All other args, to be passed through to the trigger
     #[clap(hide = true)]
     pub trigger_args: Vec<OsString>,
+
+    /// Only run a subset of the components
+    #[clap(long = "component", short = 'c')]
```

Comment:
Looks like bindle loading ignores this flag, which makes sense given this is aimed at the local dev inner loop, but means we should have a clap attribute to prevent it.